### PR TITLE
My signature

### PR DESCRIPTION
**Pull request by AKonia**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -174,8 +174,6 @@ Signed,
 
 [Andrey Unger](https://github.com/unger1984)
 
-[Denys Zelenov](https://github.com/torwalds)
-
```
Comment (nekonee) [inline index.md:178]: Please do not delete other signatures.

Comment (AKonia) [reply index.md:178]: I don't know how it happening( Maybe gh-editor bug

Comment (AKonia) [reply index.md:178]: Maybe it is merge conflict ?

Comment (AKonia) [reply index.md:178]: Because i see duplicates

**Pull request closed without merging**